zproject/default_settings.py

```diff
@@ -587,3 +587,7 @@
 # How often a client should send start notifications to the server to
 # indicate that the user is still interacting with the compose UI.
 TYPING_STARTED_WAIT_PERIOD_MILLISECONDS = 10000
+
+# We recommend to set this as False for organizations with more
+# than 100 users.
+LIMIT_PRESENCE_VISIBILITY_FOR_GUESTS = False
```

Comment:
I am not sure what should be the default value for this. Also, not sure regarding the name, do we want to use `LIMIT_PRESENCE_VISIBILITY_FOR_LIMITED_GUESTS` (we came up with "limited guest" when we were thinking of adding a separate role, but we are not doing it now). Or we can have `LIMIT_PRESENCE_VISIBILITY_FOR_RESTRICTED_GUESTS`.

Reply:
I think `CAN_ACCESS_ALL_USERS_GROUP_LIMITS_PRESENCE` would be an OK name.